Source/Plugins/Core/com.equella.core/js/tsrc/search/components/SearchResult.tsx

```diff
@@ -0,0 +1,241 @@
+      width: "88px",
+      height: "auto",
+      color: theme.palette.text.secondary,
+    },
+    itemDescription: {
+      paddingBottom: theme.spacing(1),
+    },
+    additionalDetails: {
+      flexDirection: "row",
+      display: "flex",
+      paddingTop: theme.spacing(1),
+    },
+    status: {
+      textTransform: "capitalize",
+    },
+    nested: {
+      paddingLeft: theme.spacing(4),
+    },
+  };
+});
+
+export interface SearchResultProps {
+  /**
+   * A json representation of the SearchResult data. TODO: Properly type this when search2 endpoint is merged
+   */
+  resultData: any;
+  /**
+   * Function will be invoked when the SearchResult list item is clicked
+   */
+  onClick: () => void;
+}
+export default function SearchResult({
+  resultData,
+  onClick,
+}: SearchResultProps) {
+  const classes = useStyles();
+
+  const searchResultStrings = languageStrings.searchpage.searchresult;
+
+  const {
+    name,
+    uuid,
+    description,
+    displayFields,
+    modifiedDate,
+    status,
+    displayOptions,
+    attachments,
+  } = resultData;
+
+  const [attachExapanded, setAttachExpanded] = React.useState(
+    displayOptions.standardOpen
+  );
+
+  const handleAttachmentPanelClick = (event: SyntheticEvent) => {
+    event.stopPropagation();
+    setAttachExpanded(!attachExapanded);
+  };
+
+  const thumbnail = (disableThumb: boolean) => {
+    if (attachments.length > 0 && !disableThumb) {
+      return (
+        <>
+          <img
+            className={classes.thumbnail}
+            src={attachments[0].links.thumbnail}
+            alt={attachments[0].description}
+          />
+        </>
+      );
+    } else {
+      return (
+        <Subject className={classes.placeholderThumbnail} fontSize="large" />
+      );
+    }
+  };
+
+  const itemMetadata = (
+    <div className={classes.additionalDetails}>
+      <Typography component="span" variant="body1" className={classes.status}>
+        {status}
+      </Typography>
+      <Divider
+        flexItem
+        component="span"
+        variant="middle"
+        orientation="vertical"
+      />
+      <Typography component="span" variant="body1">
+        {searchResultStrings.dateModified}&nbsp;
+        <DateDisplay displayRelative date={new Date(modifiedDate)} />
+      </Typography>
+    </div>
+  );
+
+  const customDisplayMetadata = displayFields.map((element: any) => {
+    return (
+      <>
+        <ListItem disableGutters dense>
+          <Typography
+            component="span"
+            variant="body2"
+            className={classes.heading}
+            color="textPrimary"
+          >
+            {element.name}
+          </Typography>
+          <Typography
+            component="span"
+            variant="body2"
+            className={classes.inline}
+            color="textPrimary"
+            dangerouslySetInnerHTML={{ __html: element.html }}
+          ></Typography>
+        </ListItem>
+      </>
```

Comment:
There is a single root tag, `ListItem` here, a wrapping fragment isn't needed.

```suggestion
      <ListItem disableGutters dense>
        <Typography
          component="span"
          variant="body2"
          className={classes.heading}
          color="textPrimary"
        >
          {element.name}
        </Typography>
        <Typography
          component="span"
          variant="body2"
          className={classes.inline}
          color="textPrimary"
          dangerouslySetInnerHTML={{ __html: element.html }}
        ></Typography>
      </ListItem>
```